CHANGELOG.md

```diff
@@ -78,4 +78,18 @@ the following form: YYYY.0M.0D.
 
 - Designed a StockPortfolio component
 - Designed a BankAccount component
-- Designed a DNASequence component
\ No newline at end of file
+- Designed a DNASequence component
+
+## [2024.10.02]
```

Comment:
I always recommend organizing this file in reverse chronological order, so the newest changes are at the top.